.gitignore

```diff
@@ -22,7 +22,6 @@
 
 # Dependency directories (remove the comment below to include it)
 vendor/
-Godeps/
```

Comment:
why did you delete godeps?

Reply:
had to restart the pipline. found something unused.